src/ctia/flows/crud.clj

```diff
@@ -182,7 +183,8 @@
     (assoc fm
            :events
            (doall
-            (for [entity entities]
+            (for [{:keys [error] :as entity} entities
```

Comment:
perhaps it's a good time to replace that with `doseq`?

Reply:
I can not use `doseq` here, it returns nil